### DISCUSSION TITLE
Dialog box always fully opened?

### DISCUSSION THREAD

**Discussion by DeirdreYoung** [floccusaddon/floccus#1391] [Q&A] [answered]
I use manual syncing (Push up once, Pull down once) and I dont find the "closed" Options dialog useful. Is there a way to set it to always open completely?

**Comment by marcelklehr** [comment]
>  I dont find the "closed" Options dialog useful

Can you be more specific what you mean by this? What is the closed options dialog?

**Comment by DeirdreYoung** [comment]
When clicking on the Floccus icon, the dialog box that opens has options like Push up once and Pull down once hidden under a down arrow button. It saves about one cm of space and requires two clicks. I'd say either get rid of the button or have an option to have the dialog fully open on start. But it could be just me - if there's a file to edit somewhere I'll do it :)

**Reply by marcelklehr** [reply, marked as answer]
Mh, I personally like it the way it is. Push up once and pull down once are not supposed to be used all the time, so they are hidden.

**Reply by marcelklehr** [reply]
Out of curiosity, why don't you use auto-sync?

**Reply by DeirdreYoung** [reply]
I had bad experiences. I add bookmarks on two pc's, and I think manual is safer.

**Comment by DeirdreYoung** [comment]
I'm using Firefox on Windows 10. Like I said, I'd be fine changing the setting in the extension file.

**Reply by marcelklehr** [reply]
changing the setting in the extension file is not easily done as we use compiled javascript, sorry.

**Reply by DeirdreYoung** [reply]
got you. well thanks for answering - I always share my opinion and if enough users agree, it sometimes leads to good changes
